Description: Look for qdbm under $prefix/include/qdbm too.
 The Debian package ships the header files under that directory, for
 unknown reasons.
Origin: vendor
Forwarded: not-needed
Last-Update: 2010-02-21

Index: php-5.4.11/ext/dba/config.m4
===================================================================
--- php-5.4.11.orig/ext/dba/config.m4	2013-02-16 12:22:25.000000000 +0000
+++ php-5.4.11/ext/dba/config.m4	2013-02-16 12:28:14.000000000 +0000
@@ -113,6 +113,10 @@
       THIS_PREFIX=$i
       THIS_INCLUDE=$i/include/depot.h
       break
+    elif test -f "$i/include/qdbm/depot.h"; then
+      THIS_PREFIX=$i
+      THIS_INCLUDE=$i/include/qdbm/depot.h
+      break
     fi
   done
 
